docs/stencil-docs/storefront-customization/theme-accessibility.md

```diff
@@ -0,0 +1,148 @@
+
+The aria-label attribute uses an ID reference value that matches the ID element attribute to associate an element with text. Screen readers use the text of one or more identified elements, which can be seen elsewhere on the page.
+
+
+Example
+
+The example below displays how to use an aria-label for two elements considered to be one image.
+
+```json
+<div role="img" aria-label="Cart images">
+
+<img src = "giftCard.png" alt=""/>
+<img src = "Checkout.png" alt=""/>
+</div>
+```
+
+You can add the alt text to the `img` element in the image widget of Page Builder or in the `cornerstone/assets/img/` directory.
+
+Follow the step below to add alt text to an image using the control panel:
+1. Go to **Storefront** > **My theme** in the control panel, then click **Customize**.
+2. Launch Page Builder and select the image widget.
+3. Enter the alt text.
+4. Click **Save**.
+
+![Image Widget](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/accessibility_image_widget.png "Image Widget")
+
+## Text accessibility
+Text accessibility involves making content readable and understandable. You can achieve text accessibility by using the following:
+* Color contrast
+* Text headings
+* Text size and alignment
+* Text links
+
+### Color contrast
+Color contrast is an essential factor in visual accessibility. Use contrasting colors for better readability, along with keeping headings clear and concise.
+
+Pure black text on a white background can cause eye strain. Instead, try black text on light blue, pale green, or gray background. The following example demonstrates good color contrasting and has a 14.9.1 contrast ratio.
+
+Good contrast ratio
+![Good contrast](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/accessibility_good_color_contrast.png "Good contrast")
+
+It is essential to avoid colors that are too close together. For example, avoid green and black, green and gray, and green and red. Alternatively, it is good practice to use light text on dark backgrounds or dark text on light backgrounds. The example below shows poor color contrasting because the black text is on a green background. The color constrast ratio is 3.8.1. It would have been better to use white text on a dark green background.
+
+Poor contrast ratio
+![Poor contrast](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/accessibility_poor_color_contrast.png "Poor contrast")
+
+
+NOTE: According to [WCAG 2.1 (Web Content Accessibility Guidelines)](https://www.w3.org/TR/WCAG21/), your eCommerce website must have a minimum color contrast ratio of at least 4.5.1 for standard text and 3:1 for large text.
```

Comment:
Remove - NOTE: According to [WCAG 2.1 (Web Content Accessibility Guidelines)](https://www.w3.org/TR/WCAG21/), your eCommerce website must have a minimum color contrast ratio of at least 4.5.1 for standard text and 3:1 for large text.